FlowCal/mef.py

```diff
@@ -72,14 +80,22 @@ def clustering_gmm(data,
     samples of each group, and used as initial conditions for the GMM EM
     algorithm.
 
-    `clustering_gmm` internally uses `GMM` from the ``scikit-learn``
-    library, with full covariance matrices for each cluster and a fixed,
-    uniform set of weights. This means that `clustering_gmm` implicitly
-    assumes that all bead subpopulations have roughly the same number of
-    events. For more information, consult ``scikit-learn``'s documentation.
+    `clustering_gmm` internally uses a `GaussianMixture` object from the
+    ``scikit-learn`` library (``GMM`` if ``scikit-learn``'s version is
+    lower than 0.18), with full covariance matrices for each cluster. For
+    more information, consult ``scikit-learn``'s documentation.
 
     """
 
+    # Initialize min_covar parameter
+    # Parameter is initialized differently depending on scikit's version
+    if min_covar is None:
+        if packaging.version.parse(sklearn.__version__) \
+                >= packaging.version.parse('0.18'):
+            min_covar = 1e-3
+        else:
+            min_covar = 5e-5
+
```

Comment:
Why do you need different `min_covar` values for different versions of `sklearn`? Did you come up with the new value for `sklearn >= 0.18` empirically?

Reply:
Just modified the docstring to clarify this.